src/ExRender.hs

```diff
@@ -44,11 +48,17 @@ instance Default ExCabalEnv where
             (CompilerId GHC ver) -> ver
             x -> error $ "Unsupported compiler " ++ show x
         , exCopyright = vcat
-            [ "# Copyright 2015 Mykola Orliuk <virkony@gmail.com>"
+            [ "# Copyright " <> integer currentYear
+                             <> " Mykola Orliuk <virkony@gmail.com>"
             , "# Distributed under the terms of the GNU General Public License v2"
             ]
         , exBugsTo = "virkony@gmail.com"
         }
+      where
+        -- unsafePerformIO is fine here, we just care about the year
+        (currentYear, _, _) = toGregorian
+                                . utctDay
+                                $ unsafePerformIO getCurrentTime
```

Comment:
The whole point of `ExCabalEnv` is to provide input to renderer. Why either year or `exCopyright` generation be in `Main.hs` inside of `IO` monad?

Maybe we can add an option for changing whole copyright year and mentioned person.

Reply:
This was a quick fix, I didn't see the point in making it more complicated and referential transparency is rather unimportant wrt the __year__.

A more complicated solution would be possible. But I am not sure I am going to implement that.

Reply:
No problem. That was a good signal to prioritize the problem of post-editing copyright. Thus I've pushed it in #28 .

Though there is a still need to double-check generated content for something like missing license. And there is a need to make it more explicit ( #13 ).